protostar/argument_parser/argument_parser_facade.py

```diff
@@ -67,6 +126,15 @@ def parse(
                 argument_name=arg.name, command_name=command.name if command else None
             )
 
+        inputs_format: Optional[InputFormat] = None
+        if hasattr(args, "inputs"):
+            inputs_format = check_inputs_format(args.inputs)
```

Comment:
[Design] It looks like a hack.

It looks like `argparse` provides a single element to the parser instead of a whole string.
I'd change `Argument::is_array` to 
```py
Argument::multiple_args_resolver: Optional[Literal["array", "raw"]] = None
```
We can't have an array of `input` arguments.  The purpose of "raw" resolver would be passing the unprocessed list as a string so that the parser in protostar_arg_types.py could process it correctly.

Reply:
I believe that since we resigned from [handle 3 different formats](https://github.com/software-mansion/protostar/pull/1140/commits/463782d991b10f40e59539fd53cd071e3a971c72), this is no longer relevant.

Reply:
The link to the commit doesn't work. `ArgumentParserFacade` still changes its behavior based on the argument name.

Reply:
[Add post parse actions](https://github.com/software-mansion/protostar/pull/1140/commits/b72b590a42eb6da25252d96740698688502390c3)